*** tests/A-09-02-M-ok.out	2021-10-17 13:13:27.000000000 +0100
--- tests/A-09-02-M-ok.outhyp	2021-10-24 00:25:41.154366538 +0100
***************
*** 25,37 ****
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: AAAAS1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
! BBBBW2|Pedraria Fonseca|Oeiras, Portugal|NORMAL|0|0|0|0
! CCCCCCCCP1|Lages do Chão|Lisboa, Portugal|NORMAL|0|0|0|0
! DDDDDDP3|ObBrian Rocks|Koln, Germany|NORMAL|0|0|0|0
! EEEEEER2|Jorge Figueiredo|Lisboa, Portugal|NORMAL|0|0|0|0
! EFFFFF4|Filomena Figueiredo|Lisboa, Portugal|NORMAL|0|0|0|0
! RRRrrrrrr|RRR|rr@rrr|NORMAL|0|0|0|0
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
--- 25,37 ----
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: AAAAS1|Toshiba|Tokyo, Japan|0.0|NORMAL|valor-compras|valor-vendas-efectuadas|valor-vendas-pagas
! BBBBW2|Pedraria Fonseca|Oeiras, Portugal|0.0|NORMAL|valor-compras|valor-vendas-efectuadas|valor-vendas-pagas
! CCCCCCCCP1|Lages do Chão|Lisboa, Portugal|0.0|NORMAL|valor-compras|valor-vendas-efectuadas|valor-vendas-pagas
! DDDDDDP3|ObBrian Rocks|Koln, Germany|0.0|NORMAL|valor-compras|valor-vendas-efectuadas|valor-vendas-pagas
! EEEEEER2|Jorge Figueiredo|Lisboa, Portugal|0.0|NORMAL|valor-compras|valor-vendas-efectuadas|valor-vendas-pagas
! EFFFFF4|Filomena Figueiredo|Lisboa, Portugal|0.0|NORMAL|valor-compras|valor-vendas-efectuadas|valor-vendas-pagas
! RRRrrrrrr|RRR|rr@rrr|0.0|NORMAL|valor-compras|valor-vendas-efectuadas|valor-vendas-pagas
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
