*** tests/A-10-16-M-ok.out	2021-11-05 13:01:34.000000000 +0000
--- tests/A-10-16-M-ok.outhyp	2021-11-15 00:21:41.220554424 +0000
***************
*** 140,146 ****
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: M2|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
- NEW|SAL|20
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
--- 140,145 ----
