*** tests/A-11-05-M-ok.out	2021-11-07 16:04:59.209146000 +0000
--- tests/A-11-05-M-ok.outhyp	2021-11-11 15:47:10.682530895 +0000
***************
*** 48,54 ****
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: GARRAFA|80|2000|VIDRO:1#ROLHA:1
  PAPEL|12|2000
  ROLHA|20|500
  VIDRO|30|500
--- 48,54 ----
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: GARRAFA|80|1000|VIDRO:1#ROLHA:1
  PAPEL|12|2000
  ROLHA|20|500
  VIDRO|30|500
