*** tests/A-12-05-M-ok.out	2021-11-12 10:11:14.000000000 +0000
--- tests/A-12-05-M-ok.outhyp	2021-11-15 00:21:44.024655146 +0000
***************
*** 55,62 ****
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Saldo disponível: 0
! Saldo contabilístico: 200
  Menu Principal
  1 - Abrir
  2 - Guardar
--- 55,62 ----
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Saldo disponível: 3000
! Saldo contabilístico: 0
  Menu Principal
  1 - Abrir
  2 - Guardar
