*** tests/A-12-04-M-ok.out	2021-11-07 16:04:59.231834000 +0000
--- tests/A-12-04-M-ok.outhyp	2021-11-11 11:26:08.337480329 +0000
***************
*** 62,70 ****
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Saldo disponível: -60000
! Saldo contabilístico: -60000
! Menu Principal
  1 - Abrir
  2 - Guardar
  3 - Mostrar Data
--- 62,68 ----
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Menu Principal
  1 - Abrir
  2 - Guardar
  3 - Mostrar Data
