*** tests/A-15-04-M-ok.out	2021-10-31 18:58:50.000000000 +0000
--- tests/A-15-04-M-ok.outhyp	2021-11-15 00:21:46.424741595 +0000
***************
*** 81,87 ****
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: M1|Rohit Figueiredo|New Delhi, India|NORMAL|900|0|0|0
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
--- 81,87 ----
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: M1|Rohit Figueiredo|New Delhi, India|NORMAL|0|0|90|0
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
