*** tests/A-16-02-M-ok.out	2021-11-12 12:09:49.867245230 +0000
--- tests/A-16-02-M-ok.outhyp	2021-11-12 12:15:01.735450724 +0000
***************
*** 94,100 ****
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: S1|Toshiba|Tokyo, Japan|NORMAL|250|0|25|25
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
--- 94,100 ----
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: S1|Toshiba|Tokyo, Japan|NORMAL|250|0|25|0
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
