*** tests/A-20-08-M-ok.out	2021-11-11 10:18:50.000000000 +0000
--- tests/A-20-08-M-ok.outhyp	2021-11-15 00:21:51.756934397 +0000
***************
*** 127,133 ****
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: M1|Rohit Figueiredo|New Delhi, India|ELITE|25200|0|2800|2520
  M2|Toshiba|Tokyo, Japan|ELITE|28000|0|2800|2800
  S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
  Gestão de Parceiros
--- 127,133 ----
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: M1|Rohit Figueiredo|New Delhi, India|ELITE|25200|0|2800|2800
  M2|Toshiba|Tokyo, Japan|ELITE|28000|0|2800|2800
  S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
  Gestão de Parceiros
